README.md

```diff
@@ -6,7 +6,10 @@ peer-id JavaScript implementation
 
 # Description
 
-A IPFS Peer Id is based on a sha256 has of the peer public key, using [multihash](https://github.com/jbenet/multihash)
+A IPFS Peer Id is based on a sha256 hash of the peer public key, using [multihash](https://github.com/jbenet/multihash)
+
+The public key is currently a base64 encoded string of a protobuf of an RSA DER buffer. This uses a node buffer to pass the base64 encoded public key protobuf to the multihash for ID generation. 
```

Comment:
@richardlitt as a note, this information will be valuable for core api documentation, because @nginnever had to change it to follow go-ipfs and be able to generate the same multihash